keyboards/handwired/uthol/readme.md

```diff
@@ -0,0 +1,26 @@
+Uthol
```

Comment:
Please use the default template: 

https://raw.githubusercontent.com/qmk/qmk_firmware/master/data/templates/avr/readme.md

Reply:
Updated readme to use standard template.